code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -649,6 +652,33 @@
 		mytray.adjustWater(-round(chems.get_reagent_amount(type) * 0.5))
 		mytray.adjustWeeds(-rand(1,3))
 
+/datum/reagent/chlorine/expose_obj(obj/O, reac_volume)
+	if((!O) || (!reac_volume))
+		return 0
+	var/temp = holder ? holder.chem_temp : T20C
+	O.atmos_spawn_air("chlorine=[reac_volume/2];TEMP=[temp]")
+
+/datum/reagent/chlorine/expose_turf(turf/open/T, reac_volume)
+	if(istype(T))
```

Comment:
Avoid single letter vars, changing obj/O to obj/exposed_object shouldn't mess anything up i think, same with turf/open/T